recipes/transindel/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "transindel" %}
+{% set version = "1.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "004eacdc6b61d9e3cbb8837e6faa2016a1df5e963bbbf59843b5c31756b4b3ee" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/cauyrd/transIndel/archive/v{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
```

Comment:
Using a Jinja2 variable here makes the recipe unnecessarily complicated. I recommend writing `sha256:` directly. In general, I’d argue that using a Jinja2 variable is only worth it if it is used at least twice.